src/modules/flow/iio/nodes.c

```diff
@@ -25,6 +25,21 @@
 
 #include <sol-iio.h>
 
+#define ADD_CHANNEL_AXIS(_axis, _nodename) \
```

Comment:
Guess that you could also use this for color node type, couldn't you? It would be good to think of a generic name for `axis` and `colour`, though =D